public/docs/dart/latest/quickstart.jade

```diff
@@ -204,53 +189,109 @@ p.
     the following code,
     which loads <code>main.dart</code> and instantiates the my-app component:
 
-  pre.prettyprint.linenums
-    code.
-        &lt;!doctype html>
-        &lt;html>
-          &lt;head&gt;
-            &lt;title&gt;Angular 2 Quickstart&lt;/title&gt;
-          &lt;/head&gt;
-          &lt;body>
-            &lt;my-app>&lt;/my-app>
-
-            &lt;script type="application/dart" src="main.dart">&lt;/script>
-            &lt;script src="packages/browser/dart.js">&lt;/script>
-          &lt;/body>
-        &lt;/html>
-
-// STEP 6 - Build and run ##########################
+  code-example(language="html" format="linenums").
+    &lt;!doctype html>
+    &lt;html>
+      &lt;head&gt;
+        &lt;title&gt;Angular 2 Quickstart&lt;/title&gt;
+      &lt;/head&gt;
+      &lt;body>
+        &lt;my-app>&lt;/my-app>
+
+        &lt;script type="application/dart" src="main.dart">&lt;/script>
+        &lt;script src="packages/browser/dart.js">&lt;/script>
+      &lt;/body>
+    &lt;/html>
+
+//- STEP 6 - Run ##########################
 .l-main-section
 
-  h2#section-angular-run-app 6. Build and run your app
+  h2#section-angular-run-app 6. Run your app
 
   p.
-    You have several options for running your app.
-    The quickest and easiest, for now,
-    is to open your project in <b>Dart Editor</b>,
-    right-click <b>web/index.html</b>,
-    and choose <b>Open in Dartium</b>.
-    This starts a web server
-    and opens your app in an experimental browser that contains the Dart VM.
-
-  // TODO: screenshot? embedded app?
+    You have a few options for running your app.
+    One is to launch a local HTTP server
+    and then view the app in
+    <a href="https://www.dartlang.org/tools/dartium/">Dartium</a>.
+    You can use whatever server you like, such as Python's SimpleHTTPServer.
 
   p.
     Another option is to build and serve the app using <code>pub serve</code>,
-    and then run it by visiting <b>http://localhost:8080</b> in a browser.
-    Generating the JavaScript takes a few seconds when you first visit the page,
-    and the generated JavaScript is currently large.
-    The generated JavaScript will be smaller once
-    Angular's transformer becomes available.
+    and then run it by visiting <b>http://localhost:8080</b> in any modern browser.
```

Comment:
Can we put `pub serve` as a first option. IMHO, it is the more canonical way to do this and it supports both Dartium and any browser (thanks to the browser package).